Documentation/connectors/ldap.md

```diff
@@ -284,3 +284,42 @@ connectors:
 If the search finds an entry, it will attempt to use the provided password to bind as that user entry.
 
 [openldap]: https://www.openldap.org/
+
+## Example: Searching a Active Directory server with groups
+
+The following configuration will allow the LDAP connector to search a Active Directory using an LDAP filter.
+
+```yaml
+
```

Comment:
[nit] extra newline at the top and the bottom

Reply:
Thanks, fixed.